threetenabp/build.gradle

```diff
@@ -27,10 +22,6 @@ android {
     targetCompatibility JavaVersion.VERSION_1_7
   }
 
-  dexOptions {
```

Comment:
Removed this since it got also removed in other square libs.

Reply:
@vanniktech @JakeWharton Why was this removed? Did something change in the docs? https://google.github.io/android-gradle-dsl/current/com.android.build.gradle.internal.dsl.DexOptions.html#com.android.build.gradle.internal.dsl.DexOptions:preDexLibraries

Reply:
Is there a reason why this was removed?